.env.sample

```diff
@@ -0,0 +1,3 @@
+PORT=
```

Comment:
PORT degiskeni kullaniliyor mu? Yoksa silinmesi lazim

Reply:
Abi bırası port kullandık belirmet için yazdık. Dileyen kişi istediği port numarası verebilir.